core/src/main/java/org/apache/gravitino/Configs.java

```diff
@@ -399,4 +399,11 @@ private Configs() {}
           .stringConf()
           .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
           .createWithDefault("caffeine");
+
+  public static final ConfigEntry<String> JOB_STAGING_DIR =
```

Comment:
So this configuration is set at the server level? Should we consider supporting it at the metalake level? (That means each metalake has its own staging dir)

Reply:
My design is that: this is a root dir, each metalake can have a sub-directory. Making this a metalake-level configuration can only be achieved in the metalake property, and also makes things complicated.